api/routes_rtfsp.go

```diff
@@ -567,7 +567,8 @@ func (api *API) createHostedIPFSNetworkEntryInDatabase(c *gin.Context) {
 		return
 	}
 	logger.WithField("user", username).Info("network added to user")
-	if len(users) > 0 {
+	// if there is only 1 user, this next part is covered by the previous logic
+	if len(users) > 1 {
```

Comment:
`users` always includes the user (https://github.com/RTradeLtd/Temporal/blob/acc6b41ef7eb70009b1ae714f64c7992391296da/api/routes_rtfsp.go#L550) - if anything, the previous logic should be removed in favour of the user iteration

Reply:
yes the previous logic should be removed